clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -1447,6 +1451,7 @@ public void testListConsumerGroupsWithStatesOlderBrokerVersion() throws Exceptio
         }
     }
 
+    @Deprecated
```

Comment:
nit: feels like overkill to deprecate test cases. Since they reference RETRIES directly, I don't think we need to worry about them not getting removed

Reply:
I think the deprecation warning was part of a scheme to mark which tests should be rewritten: https://github.com/apache/kafka/pull/8864/files#r439685696

Reply:
Fair enough.